krux_boto/boto.py

```diff
@@ -216,3 +256,51 @@ def wrapper(*args, **kwargs):
             return wrapper
 
         return attr
+
+class Boto(BaseBoto):
+
+    # All the hard work is done in the superclass. We just need to use the
+    # resulting object to initialize a session properly.
+    def __init__(self, *args, **kwargs):
+        # Call to the superclass to resolve.
+        super(Boto, self).__init__(*args, **kwargs)
+
+        # access the boto classes via the object. Note these are just the
+        # classes for internal use, NOT the object as exposed via the CLI
+        # or the objects returned via the get_boto* calls
+        self._boto = boto
+
+        # This sets the log level for the underlying boto library
+        get_logger('boto').setLevel(self._boto_log_level)
+
+
+class Boto3(BaseBoto):
+
+    # All the hard work is done in the superclass. We just need to use the
+    # resulting object to initialize a session properly.
+    def __init__(self, *args, **kwargs):
```

Comment:
Instead of taking arbitrary number of arguments via `*args` and `**kwargs`, we should write each individual ones out for the ease of use and clarity.